image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,49 @@
+# Amazon EC2 Image
+
+Owner: *None*
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which can only be created
+inside EC2. An image in a standard format (ova, vmdk, vhd/x, or raw) must be
+imported from S3 storage. *osbuild-composer* generates this image type in the
+VHDX format, because it is fairly modern, widely used, and uses less space than
+RAW images.
+
+The default image size is 6 GB, unless it has to be larger due to installed
```

Comment:
Hm, I don't understand. The deafult image size is fixed at 6GB, we never automatically make it larger do we?

Reply:
I was wondering about it too. Lorax does that, I guess we should also implement it.

Reply:
This was aspirational :wink: 

Changed it until we actually implement that, of course.